examples/tendermint.rs

```diff
@@ -0,0 +1,230 @@
+extern crate mariadb_proxy;
+
+extern crate abci;
+extern crate byteorder;
+extern crate env_logger;
+extern crate futures;
+extern crate futures_util;
+extern crate hyper;
+#[macro_use]
+extern crate log;
+extern crate mysql;
+extern crate sqlparser;
+extern crate tokio;
+
+use abci::*;
+use env_logger::Env;
+use futures_util::future::FutureExt;
+use hyper::{
+    body::Body,
+    client::{Client, HttpConnector},
+};
+use mysql as my;
+use std::env;
+// use mysql_async;
+use mariadb_proxy::{
+    packet::{Packet, PacketType},
+    packet_handler::PacketHandler,
+};
+use sqlparser::{dialect::GenericDialect, parser::Parser};
+
+// Convert incoming tx data to Sql string
+fn convert_tx(tx: &[u8]) -> String {
+    let sql = String::from_utf8(tx.to_vec()).unwrap();
+    return sql;
+}
+
+fn run_query_sync(sql: String) {
+    let database_url = "mysql://root:devpassword@mariadb:3306/mariadb";
+    let pool = my::Pool::new(database_url).unwrap();
+
+    info!("run_query_sync(): {}", sql);
+
+    if sql.len() > 0 {
+        let result = pool.prep_exec(sql, ());
+
+        match result {
+            Ok(_) => {
+                info!("Query successfully executed");
+            }
+            Err(_e) => {
+                info!("Query error: {}", _e);
+            }
+        }
+    }
+}
+
+struct AbciApp {
+    sql: String,
+}
+
+impl AbciApp {
+    fn new() -> AbciApp {
+        AbciApp {
+            sql: String::from(""),
+        }
```

Comment:
```suggestion
        Default::default()
```
(if you really want a `new`)